tests/lib/ocpp/common/test_message_queue.cpp

```diff
@@ -120,17 +120,16 @@ std::ostream& operator<<(std::ostream& os, const TestMessageType& message_type)
     return os;
 };
 
-template <> bool ControlMessage<TestMessageType>::isTransactionMessage() const {
-    return this->messageType == TestMessageType::TRANSACTIONAL ||
-           this->messageType == TestMessageType::TRANSACTIONAL_UPDATE;
+bool is_transaction_message(const TestMessageType message_type) {
+    return message_type == TestMessageType::TRANSACTIONAL || message_type == TestMessageType::TRANSACTIONAL_UPDATE;
```

Comment:
For these kind of expressions it improves readability if you add braces like
```
(message_type == TestMessageType::TRANSACTIONAL) || (message_type == TestMessageType::TRANSACTIONAL_UPDATE)
```
because sometimes the operator precedence is not that obvious